src/services_models/services_constructor.jl

```diff
@@ -311,11 +314,30 @@ function construct_service!(
     add_variables!(container, ActivePowerVariable, areas, T())
     add_variables!(container, DeltaActivePowerUpVariable, areas, T())
     add_variables!(container, DeltaActivePowerDownVariable, areas, T())
-    # TODO: Re-enable full support for AGC models
-    # add_variables!(container, AdditionalDeltaActivePowerUpVariable, areas)
-    # add_variables!(container, AdditionalDeltaActivePowerDownVariable, areas)
-    balancing_auxiliary_variables!(container, sys)
-    add_feedforward_arguments!(container, model, service)
+    add_variables!(container, AdditionalDeltaActivePowerUpVariable, areas, T())
+    add_variables!(container, AdditionalDeltaActivePowerDownVariable, areas, T())
+
+    add_initial_condition!(container, services, T(), AreaControlError())
+
+    add_to_expression!(
+        container,
+        EmergencyUp,
+        AdditionalDeltaActivePowerUpVariable,
+        areas,
+        model,
+    )
+
+    add_to_expression!(
+        container,
+        EmergencyDown,
+        AdditionalDeltaActivePowerDownVariable,
+        areas,
+        model,
+    )
+
+    add_to_expression!(container, RawACE, SteadyStateFrequencyDeviation, services, model)
+
+    # add_feedforward_arguments!(container, model, service)
```

Comment:
Is this supposed to be commented out? Deleted?

Reply:
Fixed